user_tools/src/spark_rapids_tools/tools/qualx/preprocess.py

```diff
@@ -330,7 +330,9 @@ def infer_app_meta(eventlogs: List[str]) -> Mapping[str, Mapping]:
                         )
                     toc_list.append(tmp)
 
-        if toc_list:
+        if not toc_list:
+            raise ValueError(f'No CSV files found for: {ds_name}')
+        else:
```

Comment:
Lets set the standard:

- raising an error: it means something is really bad. In that case do we fallback to legacy Speedup?
- Return a default DF with speedup 1.0: this is the case when the QualX says that there is no speedup because there are ceratin features, files/columns I cannot read/empty. Then the speedup is 1.0

For the case when it is "not toc_list", do we want to raise an error? or return 1.0 after logging that files were missing?
CC: @eordentlich and @parthosa